src/js/edu-benefits/5490/config/form.js

```diff
@@ -73,7 +75,14 @@ const formConfig = {
   chapters: {
     applicantInformation: {
       title: 'Applicant Information',
-      pages: {}
+      pages: {
+        applicantInformation: applicantInformation([
+          'relativeFullName',
```

Comment:
if relative == dependent? Then yes that makes sense. But shouldn't it be `relativeFirstName` etc?